### PR TITLE
Translate src/vector

### PR DESCRIPTION
**Pull request by Kegsay**
Including `src/vector/platform`.

### CODE REVIEW

**Comment by Kegsay** [comment]
Integration tests failed when GH was down:
```
The command "scripts/fetch-develop.deps.sh" failed and exited with 1 during .
```

File: src/vector/platform/ElectronPlatform.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ limitations under the License.
 
 import VectorBasePlatform from './VectorBasePlatform';
 import dis from 'matrix-react-sdk/lib/dispatcher';
+import _t from 'matrix-react-sdk/lib/languageHandler';
```
Comment (t3chguy) [inline src/vector/platform/ElectronPlatform.js]: Should it not be `import { _t } from 'matrix-react-sdk/lib/languageHandler';` ?

File: src/vector/platform/VectorBasePlatform.js [hunk 2]
```diff
@@ -18,6 +18,8 @@ limitations under the License.
 */
 
 import BasePlatform from 'matrix-react-sdk/lib/BasePlatform';
+import _t from 'matrix-react-sdk/lib/languageHandler';
```
Comment (t3chguy) [inline src/vector/platform/VectorBasePlatform.js]: same Q as above

**Comment by t3chguy** [comment]
Apart from Q, LGTM

File: src/vector/platform/ElectronPlatform.js (hunk 1 above)
Comment (Kegsay) [inline src/vector/platform/ElectronPlatform.js]: Yep, typically I had only tested this on `WebPlatform` which was the only one to import correctly. Thanks.

File: src/vector/platform/VectorBasePlatform.js (hunk 2 above)
Comment (Kegsay) [inline src/vector/platform/VectorBasePlatform.js]: Yep, typically I had only tested this on `WebPlatform` which was the only one to import correctly. Thanks.

File: src/vector/platform/ElectronPlatform.js (hunk 1 above)
Comment (t3chguy) [inline src/vector/platform/ElectronPlatform.js]: np :)

**Pull request merged**